src/test/data/draft/response/partAdmission.ts

```diff
@@ -0,0 +1,29 @@
+import { individualDetails } from '../partyDetails'
+
+export const responseDraft = {
```

Comment:
We could call it draft as it will be the only one in the file. I do not feel strongly, it feels like you followed naming started in other place.

Reply:
What would you say about making it a default export? My thinking is, if a name is irrelevant, it might be better not to have it at all than have an arbitrary one.